packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart.ts

```diff
@@ -69,7 +69,7 @@ export class ForceFunctionStartExecutor extends SfdxCommandletExecutor<string> {
     this.executionCwd = functionDirPath;
     return new SfdxCommandBuilder()
       .withDescription(nls.localize('force_function_start_text'))
-      .withArg('evergreen:function:start')
+      .withArg('run:function:start')
```

Comment:
force_function_start_warning_plugin_not_installed in the error scenario is the doc change I was talking about. This has to be updated with the new plugin information once Sonal's doc changes are ready. Can you add a story to track that? Rest looks good.